wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -0,0 +1,170 @@
+////
+// Wire
+// Copyright (C) 2023 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+enum CoreDataMessagingMigrationVersion: String, CaseIterable {
+
+    private enum Constant {
+        static let dataModelPrefix = "zmessaging"
+        static let modelDirectory = "zmessaging.momd"
+        static let resourceExtension = "mom"
+    }
+
+    // MARK: -
+
+    // Note: add new versions here in first position!
+    case version2_109 = "zmessaging2.109.0"
```

Comment:
I think in the original merge in `release` there was a new `2_210` version added. Was it dropped or will be added later in another PR (don't have the overview right now).

Reply:
added in next PR